data/story northern union.txt

```diff
@@ -82,9 +82,12 @@ mission "Meeting an Alpha 1"
 				`	(Join him and place an order)`
 				`	(Leave.)`
 					decline
-			`	"Good decision, joining me. My boss will be over any minute now." Some time passes before eventually an incredibly tall, noseless man joins you. He looks almost exactly like the descriptions of the Augments known as Alphas from centuries ago.`
-			`	"Boss, I have brought you tribute!" Peg shouts, as you feel a hard, cold object press against the side of your skull. Before you even get a chance to react, you feel a warm splattering on your cheek. You look over to see Peg, dying, with some sort of metal spike sticking out of his forehead.`
-			`	"I would appologize for the mess," the towering beast drones, "Considering I just saved your life from my incompetent underling, I'll assume you have already forgiven me." He gives off a strange huffing sound that almost seems like a chuckle, "Please, stay. I sense you aren't as useless as this deflating mess next to you."`
+			`	"Good decision, joining me. My boss will be here any minute now."` 
+			
+			`Some time passes before eventually an incredibly tall, noseless man joins you. He looks almost exactly like the descriptions of the Augments known as Alphas from centuries ago.`
+			
+			`	"Boss, I've brought you a tribute!" Peg shouts, as you feel a hard, cold object press against the side of your skull. Before you even get a chance to react, you feel something warm splattering on your cheek. You look over to see Peg, dying, with some sort of metal spike sticking out of his forehead.`
+			`	"I would appologize for the mess," the towering beast drones, "Considering I just saved your life from my incompetent underling, I'll assume you've already forgiven me." He gives off a strange huffing sound that almost seems like a chuckle, "Please, stay. I sense you aren't as useless as this deflating mess next to you."`

```

Comment:
Change:

> appologize

To:

> apologize

Reply:
Make the capital C a normal c in "Considering", as it is a continuation of the earlier quoted text, not a new sentence.